### DISCUSSION TITLE
离线安装k8s失败

### DISCUSSION THREAD

**Discussion by ttsdmmn** [labring/sealos#4960] [Q&A]
使用sealos run kubernetes.tar安装k8s，仍然需要从外网下载镜像
![1723444967849](https://github.com/user-attachments/assets/f76cf0d1-564b-4567-97b8-20091f7e2a29)

**Comment by Wjt11** [comment]
遇到相同问题，大佬解决了吗？

**Comment by fanux** [comment]
这是台干净的环境？感觉不像。 像是有 API 版本冲突。

**Comment by iris-qq** [comment]
类似的问题，有解决的么
![image](https://github.com/user-attachments/assets/5bc34786-0514-4b95-9bee-75779da8912e)
